api/src/pcapi/domain/demarches_simplifiees.py

```diff
@@ -173,3 +173,8 @@ def _find_value_in_fields(fields: list[dict], value_name: str) -> Optional[dict]
         if field["type_de_champ"]["libelle"] == value_name:
             return field["value"]
     return None
+
+
+def update_demarches_simplifiees_annotations(application_id: str, annotation_id: str, error: Exception) -> None:
```

Comment:
J'aurai bien mis la "notion" que l'annotation est un text dans le nom de la fonction et changé `error` en `message`:
```suggestion
def update_demarches_simplifiees_text_annotation(application_id: str, annotation_id: str, message: str) -> None:
```